Readme.md

```diff
@@ -14,67 +14,67 @@ $ clib install clibs/buffer
 ## API
 
 ```c
-buffer_t *
-buffer_new();
+clib_buffer_t *
```

Comment:
this would be a major breaking change, which i don't think is a good idea right now